databases/week2/part_four.sql

```diff
@@ -0,0 +1,57 @@
+DROP DATABASE IF EXISTS `hw_lesson2_part4`;
+
+CREATE DATABASE IF NOT EXISTS `hw_lesson2_part4`;
+
+USE `hw_lesson2_part4`;
+
+CREATE TABLE employee_details( 
+	id INT NOT NULL PRIMARY KEY, 
+    salary INT NOT NULL,
+    department_id INT UNIQUE NOT NULL, 
+    start_date DATE NOT NULL,
+    end_date DATE NULL,
+    status_id INT UNIQUE NOT NULL,
+    INDEX(start_date, end_date) 
+)ENGINE=INNODB DEFAULT CHARSET=UTF8MB4 COLLATE=UTF8MB4_UNICODE_CI;
+
+CREATE TABLE employees( 
+	id INT NOT NULL AUTO_INCREMENT PRIMARY KEY, 
+    name VARCHAR(255) NOT NULL, 
+    email VARCHAR(255) UNIQUE NOT NULL,
+    phone VARCHAR(255) NULL, 
```

Comment:
Think about why you want to assign 255 chars for a human name or phone which can maximum be 15 chars long. This applies everywhere you assigned inappropriate length.